### PR TITLE
Add support for minishift_home

### PR DESCRIPTION
**Pull request by robstryker**

### CODE REVIEW

File: runtimes/bundles/org.jboss.tools.rsp.server.minishift/src/main/java/org/jboss/tools/rsp/server/minishift/servertype/impl/EnvironmentUtility.java [hunk 1]
```diff
@@ -44,6 +44,11 @@ public EnvironmentUtility(IServer server) {
 			ret.put("MINISHIFT_USERNAME", user);
 			ret.put("MINISHIFT_PASSWORD", pass);
 		}
+		
+		String msHome = MinishiftPropertyUtility.getMinishiftHome(server);
+		if( msHome != null ) {
+			ret.put("MINISHIFT_HOME", msHome);
```
Comment (adietish) [inline runtimes/bundles/org.jboss.tools.rsp.server.minishift/src/main/java/org/jboss/tools/rsp/server/minishift/servertype/impl/EnvironmentUtility.java]: I would define those 2 keys as constants for improved visibility:
```
private static final String KEY_USERNAME = "MINISHIFT_USERNAME";
private static final String KEY_PASSWORD = "MINISHIFT_PASSWORD";
```

**Comment by robstryker** [comment]
testPR

**Comment by robstryker** [comment]
@adietish  can you please test this in case I missed anything here? Thanks

**Comment by robstryker** [comment]
testPR

**Comment by jrichter1** [comment]
testPR

File: runtimes/bundles/org.jboss.tools.rsp.server.minishift/src/main/java/org/jboss/tools/rsp/server/minishift/servertype/BaseMinishiftServerType.java [hunk 2]
```diff
@@ -84,6 +74,10 @@ public Attributes getOptionalAttributes() {
 					ServerManagementAPIConstants.ATTR_TYPE_STRING,
 					"A minishift profile. Default value is 'minishift'", "minishift");
 
+			attrs.addAttribute(IMinishiftServerAttributes.MINISHIFT_HOME, 
+					ServerManagementAPIConstants.ATTR_TYPE_STRING,
+					"A flag value for the --minishift-home command line launch parameter.", null);
```
[the remark this answers is not in the document]
Comment (adietish) [reply runtimes/bundles/org.jboss.tools.rsp.server.minishift/src/main/java/org/jboss/tools/rsp/server/minishift/servertype/BaseMinishiftServerType.java]: dare to correct the description from flag to env variable given that it's an env variable?

**Comment by robstryker** [comment]
testPR

**Pull request merged**